README.md

```diff
@@ -56,7 +56,7 @@ The attacker and defender should be seated opposite each other, and the area bet
 The BINMAT deck being used for gameplay ([The Binmat Deck](#the-binmat-deck)) should be shuffled, and then dealt as follows:
 
 Deal thirteen cards face down into each lane at the defender's end.
-Turn the three left decks (defender's POV) over so that their top card is visible.
+Turn the top card in the three left decks (defender's POV) over so they are visible.
```

Comment:
The top card of these decks continues to be revealed after drawing, and for the remainder of the match